ports/raspberrypi/boards/archi/mpconfigboard.mk

```diff
@@ -0,0 +1,15 @@
+USB_VID = 0x2E8A
+USB_PID = 0x1043
+USB_PRODUCT = "ARCHI"
+USB_MANUFACTURER = "NEWSAN"
+
+CHIP_VARIANT = RP2040
+CHIP_FAMILY = rp2
+
+EXTERNAL_FLASH_DEVICES = "W25Q32JVxQ"
+
+CIRCUITPY__EVE = 1
+CIRCUITPY_PICODVI = 1
+
+FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_MPU6050
+FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_Pixel_Framebuf
```

Comment:
Pixel_Framebuf depends on the LED_Animation and NeoPixel libraries, so you'll want to freeze those too, if you don't want to have the user have to add those libraries to lib/.

Alternatively, don't bother to freeze any of these, and have the user load the libraries.

Reply:
Done. Add the LED_Animation module correcting Led by LED